server/constants.py

```diff
@@ -10,6 +10,9 @@
               'revision']
 API_PREFIX = '/api'
 
+COURSE_ENDPOINT_FORMAT = '^\w+/\w+/\w+$'
+ASSIGNMENT_ENDPOINT_FORMAT = COURSE_ENDPOINT_FORMAT[:-1] + '/\w+$'
```

Comment:
We should move the regexes in converters.py here instead so there's not two different ones